packages/ui/components/DatePicker/styles.css

```
@@ -0,0 +1,53 @@
+  cursor: not-allowed !important;
+  pointer-events: none !important;
+}
+
+.react-calendar__navigation button:enabled:hover,
+.react-calendar__navigation button:enabled:focus {
+  background-color: var(--slate-200) !important;
+}
+
+.react-calendar__month-view__weekdays {
+  text-decoration-style: dotted !important;
+  text-decoration: underline;
```

Biome

[error] Unexpected shorthand property text-decoration after text-decoration-style



(lint/suspicious/noShorthandPropertyOverrides)